### PR TITLE
chore(default-orchetrator): Made docker-compose as the default orchestrator

### PR DESCRIPTION
**Pull request by Palashx**
Looked for process.env variable across the project and added the default as docker-compose where-ever necessary

### CODE REVIEW

**Comment by Palashx** [comment]
Will add test for docker-compose

File: botfront/imports/api/endpoints/endpoints.methods.js [hunk 1]
```diff
@@ -6,7 +6,9 @@ import { checkIfCan } from '../../lib/scopes';
 
 export const createEndpoints = async (project) => {
     if (!Meteor.isServer) throw Meteor.Error(401, 'Not Authorized');
-    if (!process.env.ORCHESTRATOR) return;
+    if (!process.env.ORCHESTRATOR) {
+        process.env.ORCHESTRATOR = 'docker-compose';
```
Comment (MatthieuJnon) [inline botfront/imports/api/endpoints/endpoints.methods.js]: You shouldn't set values on process.env, it's not the intended purpose.
use an intermediate variable instead like
`const orchestration = process.env.ORCHESTRATOR ? process.env.ORCHESTRATOR : 'docker-compose';`

File: botfront/imports/api/instances/instances.methods.js [hunk 2]
```diff
@@ -12,7 +12,9 @@ import { Evaluations } from '../nlu_evaluation';
 
 export const createInstance = async (project) => {
     if (!Meteor.isServer) throw Meteor.Error(401, 'Not Authorized');
-    if (!process.env.ORCHESTRATOR) return;
+    if (!process.env.ORCHESTRATOR) {
+        process.env.ORCHESTRATOR = 'docker-compose';
```
Comment (MatthieuJnon) [inline botfront/imports/api/instances/instances.methods.js]: same comment than above

Comment (Palashx) [reply botfront/imports/api/instances/instances.methods.js]: Added

**Pull request merged**